tt_metal/hw/ckernels/grayskull/metal/llk_api/llk_sfpu/llk_math_eltwise_unary_sfpu_negative.h

```diff
@@ -0,0 +1,31 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+
+#include "llk_math_eltwise_unary_sfpu_common_includes.h"
+#include "llk_math_eltwise_unary_sfpu_init.h"
+#include "llk_math_eltwise_unary_sfpu_0_param.h"
+#include "ckernel_sfpu_negative.h"
+
+namespace ckernel {
+
+// New LLK SFPU APIs
+
+template <bool APPROXIMATE>
+inline void llk_math_eltwise_unary_sfpu_negative_init() {
+    llk_math_eltwise_unary_sfpu_init<SfpuType::negative, APPROXIMATE>();
+}
+
+template <bool APPROXIMATE, DstSync Dst = DstSync::SyncFull>
```

Comment:
Didn't we get rid of all non-full in llk land? Did we re-add them into the codebase? @davorchap